FinderCompass/src/main/java/atomicstryker/findercompass/common/DefaultConfigFilePrinter.java

```diff
@@ -290,8 +292,14 @@ else if (settingList.isEmpty())
 			} else {
 				int[] ids;
 				for(int damage = 0; (ids=OreDictionary.getOreIDs(new ItemStack(pBlock, 1, damage))).length > 0; damage++){

```

Comment:
Please see comment above. This is not necessary, if we work without the placeholder -1. Normally has a block only one damage and this default is 0. If you want to use a placeholder to find all blocks of same block with different damages, it is necessary to iterate over all defined damages.
It would be logical if block with a special damage isn't registered in ore dictionary, the given set of IDs is empty. It works for a few mods without problems. But there are mods these give for any damage of a block the same set of IDs. And so I need the additional check.
It is also possible to delete the new function completely. But there is the function to search with the wildcard -1 a big set of different damages of same block at this time. And this isn't possible without the iteration over damage in ore dictionary.